evap/staff/importers/enrollment.py

```diff
@@ -594,12 +595,15 @@ def import_enrollments(
         if test_run:
             importer_log.add_success(_("The test run showed no errors. No data was imported yet."))
             msg = format_html(
```

Comment:
This doesn't need to use `format_html`, right? Then, we shouldn't use it. Repeated below.